src/main/java/org/sasanlabs/beans/LevelResponseBean.java

```diff
@@ -101,7 +117,12 @@ public void setAttackVectorResponseBeans(
 
     @Override
     public int compareTo(LevelResponseBean levelResponseBean) {
-        return LevelConstants.getOrdinal(this.level)
-                - LevelConstants.getOrdinal(levelResponseBean.getLevel());
+        ToIntFunction<String> variantOrdinal = level ->
```

Comment:
As we are not using SecureConstants so this change might not be required. please check.

Reply:
Please take a look at https://github.com/SasanLabs/VulnerableApp/pull/278#discussion_r623483099 and https://github.com/SasanLabs/VulnerableApp/pull/278#discussion_r623490100

Reply:
Resolved in 3211f26